models/SymbolMaster/SymbolMaster.js

```diff
@@ -81,10 +81,57 @@ class SymbolMaster extends Artboard {
     super.addLayer(layer);
   }
 
+  /**
+   * Update existing SymbolInstance
+   * Nested Symbols are not currently supported
+   * @property {symbolInstance} SymbolInstance
+   * @property {Object} args - overrides
+   * @property {string} args[].name - name of the override being set
+   * @property {string|Object} [args[].value] - the value set, for Layer Styles pass in object or do_objectID
+   * @property {string|Object} [args[].style] - for textStyles only, pass in TextStyle object or do_objectID
+   */
+  updateInstance(symbolInstance, args) {
+    args.forEach(arg => {
+      const overrideLayer = this.getAllLayers().find(l => l.name === arg.name);
+
+      if (overrideLayer !== undefined) {
+        const overrideName = overrideLayer.do_objectID;
+
+        symbolInstance.overrideValues
+          .filter(prop => prop.overrideName.split('_')[0] === overrideName)
+          .forEach(prop => {
+            if (prop.overrideName.includes('_stringValue')) {
+              prop.value = arg.value;
+            }
+            if (prop.overrideName.includes('_layerStyle')) {
+              prop.value = arg.value instanceof Object ? arg.value.do_objectID : arg.value;
+            }
+            if (arg.extStyle && prop.overrideName.includes('_textStyle')) {
+              prop.value = arg.style instanceof Object ? arg.style.do_objectID : arg.style.do_objectID;
+            }
+          });
+      }
+    });
+  }
+
+  /**
+   * Creates a new SymbolInstance with overrides
+   * Nested Symbols are not currently supported
+   * @property {Object} [args]
+   * @property {Object[]} [args.overrideValues] - overrides
+   * @property {string} [args.overrideValues[].name] - name of the override being set
+   * @property {string|Object} [args.overrideValues[].value] - the value set, for Layer Styles pass in object or do_objectID
+   * @property {string|Object} [args.overrideValues[].style] - for textStyles only, pass in TextStyle object or do_objectID
+   * @returns {SymbolInstance}
+   */
   createInstance(args) {
```

Comment:
I think you might need to rebase your branch, we changed `Object.assign()` to the spread operator for class instantiation

Reply:
Rebased. Should be good now?

Reply:
Looks good! 👍